Screenshoddy/UI/Preferences/Account/AccountPreferencesViewController.swift

```diff
@@ -27,6 +28,8 @@ class AccountPreferencesViewController: NSViewController {
         
         accountPreferencesView.accessIdField.delegate = self
         accountPreferencesView.secretKeyField.delegate = self
+        
+        accountPreferencesView.getBucketsButton.action = #selector(AccountPreferencesViewController.getBucketsButtonClicked)
```

Comment:
`accountPreferencesView.getBucketsButton.action = #selector(AccountPreferencesViewController.getBucketsButtonClicked)`

to 

`accountPreferencesView.getBucketsButton.action = #selector(AccountPreferencesViewController.getBucketsButtonClicked(_:))`

To make sure that sender gets passed to function.